### DISCUSSION TITLE
Dont understand what stress tesnor unit should be used for finetuning

### DISCUSSION THREAD

**Discussion by Asif-Iqbal-Bhatti** [materialyzeai/matgl#636] [General]
Hello,

I am having a problem, and the documentation is not clear to me. For training magnet potential, should the initial data for stress be in GPa or not? Here is my code. Is this right? And this is how you have done it? 

'''
atoms_list = read("MLAIMD_config.extxyz", index=":", format='extxyz')
valid_atoms = []
energies = []
forces = []
stresses = []

'''# Convert ASE stress (eV/Å^3) -> GPa
# NOTE: no sign flip here if stresses are ASE-style (positive => tension)
# list of per-frame stresses in GPa
'''

for atoms in atoms_list:
    e = atoms.info.get(energy_key)
    f = atoms.get_forces()                # (N_atoms, 3) in eV/Ang
    s = atoms.info.get(stress_key)        # eV/Ang^3 (ASE convention) multiply by -1 already
    try:
        e_val = float(e)
    except (TypeError, ValueError):
        continue
    
    f = np.array(f, dtype=float).tolist()
    s = np.array(s, dtype=float).reshape(3, 3)
    s = (s * 160.21766208).tolist()  
    
    valid_atoms.append(atoms)
    energies.append(e_val)
    forces.append(f)    # per-frame arrays
    stresses.append(s)  # per-frame (3,3)

pmg_structures = [adaptor.get_structure(a) for a in valid_atoms]

labels = {
    "energies": energies,
    "forces": [np.array(f).tolist() for f in forces],
    "stresses": stresses
}
'''

I am reading with ASE, and by default ASE converts to eV/A³, and then I convert it to GPa to fine-tune the model. I am stuck because when I do the parity plot again using ASE and the M3GNET calculator to plot with the fine-tuned model and the original model and the test data for DFT energies, the values the plot is horizontal; something is wrong in my understanding.
